charts/hdfs-namenode-k8s/README.md

```diff
@@ -28,10 +81,12 @@ HDFS `namenode` running inside a kubernetes cluster. See the other chart for
 There will be only one `namenode` instance. i.e. High Availability (HA) is not
 supported at the moment. The `namenode` instance is supposed to be pinned to
 a cluster host using a node label, as shown in the usage above. `Namenode`
-mount a local disk directory using k8s `hostPath` volume.
+mount a local disk directory using k8s `hostPath` volume. You may want to
+restrict access of `hostPath` using `pod security policy`.
```

Comment:
Not sure if this is needed, but just a thought... should we have a single section(possibly on the main page) that has all the general security considerations/recommendations?

Reply:
I think this is a good idea going forward. `spark-on-k8s` doc also took this approach after figuring out RBAC implications. And there is issue #22 for this. Let's address this as part of the issue.